packages/storybook/src/stories/grid-pagination.stories.tsx

```diff
@@ -34,7 +34,7 @@ export function PaginationDefault() {
 }
 
 export function PageSize100() {
-  const data = useData(2000, 200);
+  const data = useData(200, 20);
```

Comment:
it might be practical to have more than 2 pages, to test

Reply:
So?
```suggestion
  const data = useData(2000, 20);
```